docs/core/integration-quality-scale/rules/iqs040.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "IQS040 - The documentation includes a high-level description of the brand, product, or service it integrates with; if available, with links to the brand, product, or service website"
+---
+
+IQS040 is as follows:
+
+The documentation includes a high-level description of the brand, product, or service it integrates with; if available, with links to the brand, product, or service website
+
+## Reasoning
+
+When writing documentation for an integration, the first thing that usually comes top of mind by developers is describing what their integration does.
+While it can be very helpful for users to also know what the integration is integrating with, it is often overlooked.
+We think its a good practice to include a high-level description of the brand, product, or service that the integration is integrating with.
+
+## Example implementation
+
+```markdown
+The **my integration** integration is used to integrate with the devices of [MyCompany](https://www.mycompany.com).
+They create various smart home appliances and devices and are known for their MyProduct.
+```
+
+## Additional resources
+
+
+## Exceptions
+
+Integrations that do not integrate with a device or service can't include this, thus they are exempt.
```

Comment:
What would be a good example of such an integration? Internal integrations?

Reply:
Yep!